### PR TITLE
docs: add README.md for apisix-dashboard

### LINKED ISSUE #189: docs: add READEME.md to artifacthub.io
I want to deploy apisix-dashboard with helm, so I visit artifacthub.io to search  `apisix-dashboard`, but get the `apisix-dashboard` doesn't have READEME.md

image

Do we need to add README.md for `apisix-dashboard` ?

### PR DESCRIPTION
**Pull request by zaunist**
resolve #189 

Add README.md for apisix-dashboard chart

### CODE REVIEW

**Comment by tao12345666333** [comment]
Thanks

File: charts/apisix-dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+This chart bootstraps an apisix-dashboard deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+APISIX Dashboard requires Kubernetes version 1.14+.
+
+## Get Repo Info
+
+```console
```
Comment (juzhiyuan) [inline charts/apisix-dashboard/README.md:13]: console?

**Review by juzhiyuan — APPROVED** [review]
console is like sh?

File: charts/apisix-dashboard/README.md (hunk 1 above)
Comment (zaunist) [reply charts/apisix-dashboard/README.md:13]: 😂 Oh, this is a mistake, I will update.

Comment (zaunist) [reply charts/apisix-dashboard/README.md:13]: > console?

console is used to highlight (display to the user) a sequence of shell session commands (with prompts, not just the contents of the prompt script file).

Comment (juzhiyuan) [reply charts/apisix-dashboard/README.md:13]: get it

**Review by tokers — APPROVED** [review]

**Pull request merged**